makefile.shade

```diff
@@ -3,8 +3,19 @@ var VERSION='0.1'
 var FULL_VERSION='0.1'
 var AUTHORS='Microsoft Open Technologies, Inc.'
 
+default BASE_DIR='${Directory.GetCurrentDirectory()}'
+default TARGET_DIR='${Path.Combine(BASE_DIR, "artifacts")}'
+
 use-standard-lifecycle
 k-standard-goals
 
 #xml-docs-test .clean .build-compile description='Check generated XML documentation files for errors' target='test'
-  k-xml-docs-test
\ No newline at end of file
+  k-xml-docs-test
+
+#nuget-pack target='package'
+  @{
+  	var outputDir = Path.Combine(TARGET_DIR, "build");
+  	var existingNupkg = Directory.EnumerateFiles(outputDir).First(f => f.EndsWith(".nupkg"));
+  	var version = Path.GetFileNameWithoutExtension(existingNupkg).Substring("NuGetPackageVerifier.".Length);
+  }
+  exec program='.build/NuGet.exe' commandline='pack -nopack ${Path.Combine("src", "NuGetPackageVerifier", "NuGetPackageVerifier.nuspec")} -version ${version} -base ${Path.Combine(TARGET_DIR, "build", "NuGetPackageVerifier", "net451")} -out ${outputDir}'
```

Comment:
Does KoreBuild allow wrapping these on to a second line? If not, could you extract the paths into variables?